l10n_it_edi_extension/views/l10n_it_view.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="account_invoice_form_l10n_it_inherit_view" model="ir.ui.view">
+        <field name="name">account.move.form.l10n.it.inherit.view</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="l10n_it_edi.account_invoice_form_l10n_it" />
+        <field name="arch" type="xml">
+            <data>
+                <xpath
+                    expr="//page[@name='electronic_invoicing']//field[@name='l10n_it_edi_attachment_id']"
+                    position="attributes"
+                >
+                    <attribute name="invisible">1</attribute>
+                </xpath>
+                <xpath
+                    expr="//page[@name='electronic_invoicing']//field[@name='l10n_it_edi_attachment_id']"
+                    position="after"
+                >
+                    <label
+                        for="l10n_it_edi_attachment_id"
+                        class="oe_inline"
+                        invisible="not l10n_it_edi_attachment_id"
+                    />
+                    <div class="o_row" invisible="not l10n_it_edi_attachment_id">
+                        <field
+                            name="l10n_it_edi_attachment_id"
+                            groups="account.group_account_invoice"
```

Comment:
qua l'obiettivo è cambiare il gruppo di `l10n_it_edi_attachment_id`? In tal caso non è meglio farlo direttamente con `attribute` invece che metterlo invisibile e riaggiungerlo dopo?

Reply:
in realtà non è solo per il gruppo a è per aggiungere il pulsante per la preview